pages/common/peerflix.md

```diff
@@ -1,15 +1,23 @@
 # peerflix
 
-> Stream video or audio based torrents to your favorite player.
+> Stream video- or audio-based torrents to a media player.
 
-- Stream a torrent:
+- Stream the largest media file in a torrent given as a magnet link:
 
-`peerflix "{{magnet:?xt=urn:btih:00112233445566}}"`
+`peerflix "{{magnet:?xt=urn:btih:0123456789abcdef0123456789abcdef01234567}}"`
 
-- Stream a torrent to VLC:
+- Stream the largest file in a torrent, given as a torrent URL, to VLC:
 
-`peerflix "{{torrent url|magnet link}}" --vlc`
+`peerflix "{{http://example.net/music.torrent}}" --vlc`
```

Comment:
Yeah, having some with a concrete example and others with `torrent url|magnet link` is a good idea :D